shell/platform/windows/external_texture_gl.cc

```diff
@@ -93,8 +93,8 @@ bool ExternalTextureGL::PopulateTexture(size_t width,
   opengl_texture->target = GL_TEXTURE_2D;
   opengl_texture->name = state_->gl_texture;
   opengl_texture->format = GL_RGBA8;
-  opengl_texture->destruction_callback = (VoidCallback) nullptr;
-  opengl_texture->user_data = static_cast<void*>(this);
+  opengl_texture->destruction_callback = [](void* user_data) {};
```

Comment:
Why not just nullptr?

Reply:
I wasn’t sure if there are null checks in `embedder.cc` and Skia’s `SkImage::MakeFromTexture` but I just checked and apparently this can be just set to nullptr.